Project-Aurora/Project-Aurora/Settings/KeyboardLayoutManager.cs

```diff
@@ -750,6 +750,28 @@ public void LoadBrand(PreferredKeyboard keyboard_preference = PreferredKeyboard.
                 layoutConfigPath = Path.Combine(layoutsPath, "drevo_blademaster.json");
             else if (keyboard_preference == PreferredKeyboard.Wooting_Two)
                 layoutConfigPath = Path.Combine(layoutsPath, "wooting_two.json");
+            else if (keyboard_preference == PreferredKeyboard.Keyboard101)
+                layoutConfigPath = Path.Combine(layoutsPath, "keyboard101.json");
+            else if (keyboard_preference == PreferredKeyboard.Keyboard102)
+                layoutConfigPath = Path.Combine(layoutsPath, "keyboard102.json");
+            //keyboare 2.1
+            else if (keyboard_preference == PreferredKeyboard.Keyboard21us)
```

Comment:
Why here are you doing the languages separation? I don't see why it can't be resolved in the languages preference, and I don't think is the best practice to do languages separation in two places.

Reply:
That's just difference keyboard type name, not languages separation.